hosting/src/components/Sidebar/Sidebar.tsx

```diff
@@ -49,6 +38,30 @@ const Sidebar = ({sidebarOpen, setSidebarOpen}: SidebarProps) => {
     return () => document.removeEventListener("keydown", keyHandler);
   });
 
+  // close sidebar when page is changed or window resize
+  useEffect(() => {
+    setSidebarOpen(false);
+  }, [pathname, windowSize]);
+
+  useEffect(() => {
+    // Handler to call on window resize
+    const handleResize = () => {
+      setWindowSize({
+        width: window.innerWidth,
+        height: window.innerHeight,
+      });
+    };
+
+    // Add event listener
+    window.addEventListener("resize", handleResize);
+
+    // Call handler right away so state gets updated with initial window size
+    handleResize();
+
+    // Clean up the event listener on component unmount
+    return () => window.removeEventListener("resize", handleResize);
+  }, []); // Empty array ensures that effect is only run on mount and unmount
+
   useEffect(() => {
     localStorage.setItem("sidebar-expanded", sidebarExpanded.toString());
```

Comment:
What's the purpose of storing this item in local storage? I don't see where we're using that value retrieved from local storage.

Reply:
It looks like it is old code and is no longer used. We can remove it from there

Reply:
Done ✅ 
Thanks for the feedback @detautama